### PR TITLE
Pivot to checksum directly, and print OSTree version

### PR DESCRIPTION
**Pull request by jlebon**
See individual commit messages.

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -76,7 +76,7 @@ func Execute(cmd *cobra.Command, args []string) {
 	if len(defaultDeployment.CustomOrigin) > 0 {
 		if strings.HasPrefix(defaultDeployment.CustomOrigin[0], "pivot://") {
 			previousPivot = defaultDeployment.CustomOrigin[0][len("pivot://"):]
-			glog.Infof("Previous pivot: %s\n", previousPivot)
+			glog.Infof("Previous pivot: %s", previousPivot)
```
Comment (ashcrow) [inline cmd/root.go]: TIL `Infof` doesn't require a `\n`!

**Comment by ashcrow** [comment]
@openshift-ci-robot hello? :-)

**Comment by jlebon** [comment]
I don't think Tide is setup for this repo.

**Pull request merged**